### ISSUE TITLE
Remaining prototype tasks MIPS/ATLAS

### ISSUE THREAD

**Issue by CallmeT-ty** [powerhouse-inc/document-drive#289]
https://www.notion.so/makerdao-ses/Atlas-v2-Dashboard-Quality-Assurance-e7c3bce0e36042af99cfdc5b08dfa1df

- [ ] - Optimize Atlas Portal collapse/expand functionality 
- [ ] - Bring back the Atlas Portal sidebar with ToC
- [ ] - Mobile improvement for Atlas Portal

**Comment by CallmeT-ty** [comment]
@ryanwolhuter these are all finished right?

**Comment by ryanwolhuter** [comment]
@CallmeT-ty yes all done, the new ones are being tracked [here](https://www.notion.so/makerdao-ses/7827088d6cb4417ca067d2bb399afc1b?v=75c4db7f1ca74c7e9999d5c64cc4cfb9) so we can perhaps just keep this ticket and link to the notion page
